package.json

```diff
@@ -1,10 +1,17 @@
 {
   "name": "focusable-selectors",
+  "type": "module",
   "version": "0.7.0",
   "description": "A list of CSS selectors for focusable elements",
-  "main": "index.js",
-  "module": "index.mjs",
-  "types": "index.d.ts",
+  "main": "./index.js",
+  "module": "./index.mjs",
+  "types": "./index.d.ts",
```

Comment:
🔍 Because export maps explicitly require file paths to begin with `./`, I changed the other local paths for consistency.